concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-396	396	irritant	irritating			8.45	4.65	4.25	4.95	5.9	4.5	1.7	8	3	49832	0	107	361	4.04722951137684	0.000782868447329
+Verheyen-2019-1000-397	397	jaarlijks	annual			7.85	4.1	6.35	3.7	5	4.9	4.5	9	2	55147	0	1210	278	0.310876387183308	9.57770994325877E-05
+Verheyen-2019-1000-398	398	jaloers	yellow	YELLOW	1424	6.95	4.65	4.25	3.7	5.45	5.9	2.4	7	2	69246	0	1097	1998	1.33042978601033	0.000244337039066
+Verheyen-2019-1000-399	399	jarig	'to have one''s birthday'			3.95	5.65	6.55	4.35	5.35	5.65	5.65	5	2	30951	4	382	1068	0.416589147466834	5.06057921064206E-05
+Verheyen-2019-1000-400	400	jeugdig	youthful			9.1	5.55	5.2	4.45	3.7	5.2	5.5	7	2	19904	0	671	115	0.523165682488493	4.87346371671118E-05
+Verheyen-2019-1000-401	401	jong	young	YOUNG	1207	4.95	5.55	4.26315789473684	3.85	6	6.2	5.6	4	1	26465	8	13261	8594	11.9119821588688	0.001598467462275
+Verheyen-2019-1000-402	402	joods	jewish			9.45	3.6	5.35	3.5	2.95	5.45	3.9	5	1	16162	2	2160	918	0.663941341484769	0.000106496376598
+Verheyen-2019-1000-403	403	juist	right	CORRECT (RIGHT)	1725	4.05	4.3	5.25	4.15	6.2	3.7	5.63157894736842	5	1	34812	3	16928	8721	6.37860936046338	0.000614871275604
+Verheyen-2019-1000-404	404	kaal	bald	BALD	1125	5.65	3.25	6.25	3.2	5.05	6.75	2.7	4	1	26218	21	1594	671	1.97276029950308	0.000466249506503
+Verheyen-2019-1000-405	405	kaki	khaki	KHAKI	3750	8.95	3.35	5.52631578947368	3.8	2.15	6.45	2.75	4	2	9342	6	0	9	0.229971838938299	3.771597521146E-05
+Verheyen-2019-1000-406	406	kalm	calm	CALM	258	5.5	2.15	4.3	2.65	5.1	4.25	5.5	4	1	11875	10	2047	3170	2.83010390505845	0.000384935078842
+Verheyen-2019-1000-407	407	kansloos	prospectless			10.45	2.35	3.65	2.7	3.2	3.8	2.45	8	2	45746	0	37	131	0.162959245035873	3.71367307259001E-05
+Verheyen-2019-1000-408	408	kapot	broken	BROKEN	588	3.95	3.31578947368421	5.9	4.1	5.35	5.3	2.25	5	2	13425	3	1672	3898	9.89256544799116	0.001203523602781
+Verheyen-2019-1000-409	409	karig	scant			14.125	3.75	3.21428571428571	3.4	1.8125	4	2.94117647058824	5	2	33499	6	140	20	0.125964282759616	1.885798760573E-05
+Verheyen-2019-1000-410	410	katholiek	catholic			7.05	2.95	5.2	3.9	3.65	4.75	3.4	9	3	59741	0	2502	593	3.66413310240345	0.000409366396348
+Verheyen-2019-1000-411	411	kattig	catty			10.6	5.10526315789474	4.25	5.10526315789474	2.2	4.55	2.75	6	2	32669	7	37	18	0.098213821512021	4.05796558142284E-05
+Verheyen-2019-1000-412	412	kernachtig	pithy			11.6315789473684	4.38888888888889	3.75	5.38888888888889	2.55	3.31578947368421	3.8	10	3	96963	0	79	5	0.07456382023548	1.65865682185342E-05
+Verheyen-2019-1000-413	413	kersvers	quite new			8.5	4.75	5.4	4.55	3.15789473684211	4.05	5.35	8	2	112045	0	61	19	0.020537406226801	7.24706443450574E-06
+Verheyen-2019-1000-414	414	keurig	neat			8.4	3.4	4.42105263157895	3.9	2.95	6.05	4.75	6	2	35431	2	1613	340	0.654457356627569	4.68960240528344E-05
+Verheyen-2019-1000-415	415	kil	chilly	COLD (CHILL)	2932	9.94736842105263	3.1	3.6	3.6	2.85	5.3	2.45	3	1	5484	18	904	294	1.80023441613453	0.000411328668398
```

Comment:
The mapped concept refers to the noun, if I understand it correctly. So perhaps 1287 COLD or 2483 COLD (OF WEATHER) would be a better fit?

Reply:
I couldn't find a mapping for _chilly_ and we already have _cool_ mapped to COLD (OF WEATHER). Therefore, I will leave it unmapped.